pages/support.html

```diff
@@ -14,35 +14,40 @@ <H1>{% trans "Community Support" %}</H1>
+      {% blocktrans with mixxx_wiki_link="/wiki/" beginner_dj_links="/wiki/doku.php/beginner_dj_links" wiki_troubleshooting="/wiki/doku.php/troubleshooting" %}The <a href="{{ mixxx_wiki_link }}">Mixxx Wiki</a> has a wealth of useful documentation for users and developers. New to it all and not sure how to get started? Checkout the <a href="{{ beginner_dj_links }}">Beginner DJ Links</a>. Running into technical issues? See the <a href="{{ wiki_troubleshooting }}">Troubleshooting</a> guide. {% endblocktrans %}
     </p>
   </div>
-  <div style="clear: both;"></div>
 
   <div class="halfbox_left">
-    <h2>{% trans "Community Forums" %}</h2>
+    <h2>{% trans "User Manual" %}</h2>
     <p>
-      {% blocktrans with forums_link="/forums" %}Have a Mixxx-related question? Something other DJs can help you with? Ask in the <a href="{{ forums_link }}">Mixxx Community Forums</a>.{% endblocktrans %}
+      {% blocktrans with manual_latest_link="/manual/latest" manual_beginner_link="/manual/latest/chapters/djing_with_mixxx.html" %}The <a href="{{ manual_latest_link }}">Mixxx Manual</a> provides an overview of Mixxx's user interface, information about <b>configuration</b> and hardware, and much more. It also contains useful information for <b>beginners</b> under <a href="{{ manual_beginner_link }}">DJing with Mixxx</a>.{% endblocktrans %}
     </p>
   </div>
+  <div style="clear: both;"></div>
 
   <div class="halfbox_right">
     <h2>{% trans "Zulip Chat" %}</h2>
     <p>
-      {% blocktrans with zulip_web_link="https://mixxx.zulipchat.com/" zulip_app_link="https://zulipchat.com/apps/" %}<a href="{{ zulip_web_link }}">Mixxx's Zulip chat instance</a> can be used for support too. If you don't need support, Zulip is still a fun place to hang out with the developers and meet some fellow DJs. Join the discussion <a href="{{ zulip_web_link }} ">on the web</a> or download the <a href="{{ zulip_app_link }}">Zulip app</a> and configure it to use the server mixxx.zulipchat.com{% endblocktrans %}
+      {% blocktrans with zulip_web_link="https://mixxx.zulipchat.com/" zulip_app_link="https://zulipchat.com/apps/" %}<a href="{{ zulip_web_link }}">Mixxx's Zulip chat instance</a> is a fun place to hang out with the developers and meet some fellow DJs. Here, the Mixxx team discusses upcoming changes and programming related stuff. It can be used for support too. Join the discussion <a href="{{ zulip_web_link }} ">on the web</a> or download the <a href="{{ zulip_app_link }}">Zulip app</a> and configure it to use the server mixxx.zulipchat.com. {% endblocktrans %}
     </p>
-  </div>
+  </div> 
 
-  <div style="clear: both;"></div>
```

Comment:
This should remain to keep the correct spacing. (Basically, you need the `clear: both;` before any `halfbox_left` DIV.

Reply:
It looks like we do not need these lines at all. 
At least they make no differences on my system here. 
But I will restore them, who knows.